pkg/policy/api/groups.go

```diff
@@ -80,10 +79,6 @@ func (group *ToGroups) GetCidrSet() ([]CIDRRule, error) {
 		ips = append(ips, awsIPs...)
 	}
 
-	// Sort IPS to have always the same result and do not update policies if it
-	// is not needed.
-	sort.Slice(ips, func(i, j int) bool {
-		return bytes.Compare(ips[i], ips[j]) < 0
-	})
-	return IPsToCIDRRules(ips), nil
+	resultIps := KeepUniqueIPs(ips)
```

Comment:
What happened here?

Reply:
A new helper was used from the FQDN package.